packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -771,7 +771,7 @@ class SliverAppBar extends StatefulWidget {
   ///
```

Comment:
Can you also replace the line "For example:" with "## Sample code"? That way, the code below should be automatically analyzed during presubmit and we can catch if this code snippet goes out of date in the future.